app/src/main/res/layout/item_status.xml

```diff
@@ -17,7 +17,7 @@
         android:layout_marginStart="14dp"
         android:layout_marginTop="@dimen/status_reblogged_bar_padding_top"
         android:layout_marginEnd="14dp"
-        android:drawableStart="@drawable/ic_reblog_18dp"
+        android:drawableStart="@drawable/ic_reblog_24dp"
```

Comment:
Why is this switching to the bigger icon? It causes the effective left margin of the text to change.

Reply:
Also, you could copy `ic_reply_24dp.xml` to ic_reply_18dp.xml`, and then use that for replies.

Reply:
The size is determined by the component (height of TextView).

Already before: The text behind the drawable is not alligned with the name or the content below. This does not change here.

So I could leave it as 18 but that would not change it graphically.